metapackages/auto-instrumentations-node/src/utils.ts

```diff
@@ -149,6 +149,12 @@ export function getNodeAutoInstrumentations(
   const enabledInstrumentationsFromEnv = getEnabledInstrumentationsFromEnv();
   const disabledInstrumentationsFromEnv = getDisabledInstrumentationsFromEnv();
 
+  if (enabledInstrumentationsFromEnv && disabledInstrumentationsFromEnv) {
```

Comment:
Can you add this clarification to the [docs](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/metapackages/auto-instrumentations-node)? This behaviour was not clear (at least to me)

Reply:
Good callout! I've updated the README to match what will live in public docs. Also behavior probably wasn't clear because... it wasn't accurate 😅  I had some things mixed up when I was working on this but should be fixed up now!

Reply:
nit (<--- only a nit because I don't want to derail this PR): I wouldn't say no to a change that drops this diag.debug completely. I think the behaviour (start with enabled-list, then subtract disabled-list) isn't surprising, and I don't think it is unreasonable usage to have both envvars set.   Keeping it is fine, too. :)

Reply:
I'm totally onboard with dropping the debug altogether. I only added it when I was surprised by the disabling behavior. Now that I realize my brain was somewhere else that day, I agree that it doesn't help much. Will drop.